### ISSUE TITLE
document that "config set substitute-path" supports empty string substitution

### ISSUE THREAD

**Issue by oakad** [go-delve/delve#3332]
Follow up to  #3088

The only way to supply per-project settings to `dlv` appears to be `--init` option. This option takes a file containing cli commands.

In some cases, it is not possible to run the `dlv` from the project working directory, despite what  some people may think ([#3088#discussion_r938455671](https://github.com/go-delve/delve/pull/3088#discussion_r938455671)).

Therefor there either must be a way to set  an "empty" path substitution via the interactive command (init file, actually) or an additional command shall be provided to change the working directory from inside `dlv`.

The primary use case for this is using `dlv` with `bazel`, where path substitution map must be generated by a helper rule and provided to `dlv` invocation.

**Comment by aarzilli** [comment]
> In some cases, it is not possible to run the dlv from the project working directory

why not?

**Comment by oakad** [comment]
Because of the mess bazel rules are. :-)

A good feature for dlv will be to support an explicit `directory` statement, akin to how gdb supports it.

**Comment by aarzilli** [comment]
Empty strings can already be passed to `config substitute-path` as follows:

```
(dlv) config substitute-path "" nothing-into-something
(dlv) config substitute-path something-into-nothing ""
```

I'm going to keep this open to track better documentation for this.

**Comment by aarzilli** [comment]
Fixed by 13ad7dc1d581b86d7000f6fc4bd37309c30ec480
